src/main/java/com/redhat/devtools/lsp4ij/settings/ui/JsonTextField.java

```diff
@@ -86,8 +87,8 @@ public void setJsonFilename(@NotNull String jsonFilename) {
 
     // Proxy some simple accessors to the editor text field
 
-    public void setText(@NotNull String text) {
-        editorTextField.setText(text);
+    public void setText(@Nullable String text) {
```

Comment:
Made this more flexible with regard to `null` values which result in a valid default JSON document for the text editor.